db/migrate/20171112174357_create_katello_product_content_content.rb

```diff
@@ -0,0 +1,13 @@
+class CreateKatelloProductContentContent < ActiveRecord::Migration
+  def change
+    create_table :katello_product_content_contents do |t|
+      t.string :content_id
```

Comment:
I assume this content_id is the candlepin identifier?  I think renaming to be more clear if so would be helpful, we use cp_id in a lot of places to indicate that

Reply:
That's not the candlepin UUID - but the actual content_id taken from the Content.java in candlepin. It's what we pass from the UI right into the backend. I did have cp_id in there, so it's funny you mention that, but I removed it because it was not the actual cp_id and it looks like we don't need to care about it.

candlepin=# SELECT uuid,content_id,name,label from cp2_content limit 1;
               uuid               |  content_id   |   name   |                   label                   
----------------------------------+---------------+----------+-------------------------------------------
 4028f9e65f2c601f015f2cc03f910004 | 1508284317562 | testrepo | Default_Organization_testproduct_testrepo

Reply:
maybe naming it 'candlepin_content_id' would make it clearer?